README.md

```diff
@@ -128,6 +126,19 @@ To save DataFrame as avro you should use the `save` method in `AvroSaver`. For e
 ```scala
 scala> AvroSaver.save(myRDD, "my/output/dir")
 ```
+
+Avro schema's field 'doc' will be preserved in DataFrame's schema metadata.
+
+To include aliases column in scheme invoke
```

Comment:
To include extra columns when aliases are specified in the schema, set `withAlias` to `true`, as follows:

I would also call the property `withAliases`, since we handle more than one.